rts/lra/lra-client/src/main/java/io/narayana/lra/client/LRAClient.java

```diff
@@ -234,7 +236,6 @@ public URL startLRA(String clientID) throws GenericLRAException {
         return startLRA(clientID, 0L);
```

Comment:
I think `public` methods should have a javadoc describing a purpose of the method.

Reply:
This class implements LRAClientAPI and the javadoc is in the interface class - I would rather not duplicate the same javadoc so as to avoid the descriptions diverging.